_maps/map_files/BallMap/summer_ball.dmm

```diff
@@ -0,0 +1,3184 @@
+"lx" = (/obj/structure/ladder/unbreakable/hole{height = 1; id = "WelcomeToTheThroneOfDeath1"},/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"ly" = (/obj/machinery/button/door{desc = "For use when the time is right."; id = "HouseOfSpooksGrandOpening"; name = "Open-says-me"},/turf/open/floor/engine,/area/awaymission/beach)
+"lz" = (/turf/open/floor/wood{icon_state = "wood-broken5"},/area/awaymission/beach)
+"lA" = (/turf/open/floor/wood{icon_state = "wood-broken7"},/area/awaymission/beach)
+"lB" = (/turf/open/floor/wood{icon_state = "wood-broken"},/area/awaymission/beach)
+"lC" = (/obj/structure/spider/stickyweb,/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"lD" = (/turf/open/floor/wood{icon_state = "wood-broken2"},/area/awaymission/beach)
+"lE" = (/obj/effect/decal/cleanable/cobweb/cobweb2,/obj/structure/spider/stickyweb,/obj/effect/decal/remains/human,/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"lF" = (/obj/effect/decal/cleanable/cobweb,/obj/structure/spider/stickyweb,/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"lG" = (/obj/effect/turf_decal/trimline/red/filled/line{dir = 10},/obj/structure/closet/secure_closet/hos{anchored = 1; max_integrity = 2500},/turf/open/floor/plasteel/dark,/area/security/brig/upper)
+"lH" = (/turf/open/transparent/openspace,/area/shuttle/abandoned{name = "NTSS Spirit of Freedom Upper Deck"})
+"lI" = (/obj/effect/turf_decal/trimline/red/filled/line{dir = 4},/obj/structure/closet/secure_closet/brig{name = "Prisoner Locker"},/turf/open/floor/plasteel/dark,/area/security/brig)
+"lJ" = (/obj/effect/turf_decal/trimline/blue/filled/warning{dir = 1},/obj/structure/chair{dir = 1},/turf/open/floor/plasteel/white,/area/medical/medbay/lobby)
+"lK" = (/obj/structure/railing/corner{icon_state = "railing_corner"; dir = 4},/turf/open/floor/plating/beach/sand,/area/awaymission/beach)
+"lL" = (/obj/structure/spider/stickyweb,/obj/effect/decal/remains/human,/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"lM" = (/obj/structure/spider/stickyweb,/mob/living/simple_animal/hostile/retaliate/bat{desc = "A fierce companion for any person of power, this spider has been carefully trained by Nanotrasen specialists. Its beady, staring eyes send shivers down your spine."; emote_hear = list("chitters"); faction = list("spiders"); harm_intent_damage = 3; health = 200; icon_dead = "guard_dead"; icon_gib = "guard_dead"; icon_living = "guard"; icon_state = "guard"; maxHealth = 250; max_co2 = 5; max_tox = 2; melee_damage_lower = 15; melee_damage_upper = 20; min_oxy = 5; movement_type = 1; name = "Sergeant Araneus"; real_name = "Sergeant Araneus"; response_help_continuous = "pets"; response_help_simple = "pet"; turns_per_move = 10},/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
```

Comment:
why the fuck are you using a bat as the base creature here when what you clearly want is Sergeant Araneus?

Reply:
because sergeant araneus is actually a bat for some reason.

Reply:
that got fixed jeril

Reply:
it's in as a reference then

Reply:
> because sergeant araneus is actually a bat for some reason.

what the damn hell